string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,11 @@
+/* exported isVowel */
+// define function
+function isVowel(char) {
+  // if the lowercase value of the character is equal to any of the listed vowels,
+  if (char.toLowerCase() === 'a' || char.toLowerCase() === 'e' || char.toLowerCase() === 'i' || char.toLowerCase() === 'o' || char.toLowerCase() === 'u') {
+    // push out a value of true
+    return true;
+  }
+  // push out a value of false
```

Comment:
This pseudocode would be better as "otherwise, push out a value of false". Remember, pseudocode should be a list of instructions. I suggest you write your pseudocode as a list so you can easily read through it to make sure your wording is intuitive.